Casks/microsoft-word.rb

```diff
@@ -1,17 +1,21 @@
-cask "microsoft-word" do
-  if MacOS.version <= :el_capitan
+eask "microsoft-word" do
```

Comment:
```suggestion
cask "microsoft-word" do
```

Not sure what happened here 😅